Week-1/week1_assignment_1_2_cattyBoard/cattyBoard.html

```diff
@@ -0,0 +1,197 @@
+      height: 80px;
+      display: flex;
+      display: table-cell;
+      vertical-align: top;
+    }
+  </style>
+  <title>Catty Board</title>
+  <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+<body>
+  <header>
+    <div class="chartimg"><img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" height=150em width=150em></div>
+    <div class="charttext">
+      <H3>ChattyBoard Top 100 Single Charts (11.06.36)</H3>
+      <p id="artistName"> Unkown Artist </p>
+      <p id="detail">2016.Charts.100 songs</p>
+      <p>
+        <Button><i class="la la-play"></i></Button> Play all
+        <Button><i class="la la-plus"></i></Button> Add to
+        <Button><i class="la la-ellipsis-h"></i></Button> More
+      </P>
+    </div>
+  </header>
+  <section>
+    <nav>
+      <ul>
+        <li><Button><i class="la la-bars"></i></Button></li>
+        <li><Button><i class="la la-home"></i></Button></li>
+        <li><Button><i class="la la-search"></i></Button></li>
+        <li><Button><i class="la la-volume-up"></i></Button></li>
+        <li><Button><i class="la la-user"></i></Button></li>
+        <li><Button><i class="la la-spotify"></i></Button></li>
+        <li><Button><i class="la la-cog"></i></Button></li>
+        <li><Button><i class="la la-soundcloud"></i></Button></li>
+      </ul>
+    </nav>
+    <article>
+      <table id="playlist">
+        <tr>
+          <td class ="index">1</td>
+          <td>One Dance</td>
+          <td>Crake feat Catkid & Cyla</td>
+          <td>2:24</td>
+          <td><Button>CATTY CLOUD SYNC</Button></td>
+        </tr>
+        <tr>
+          <td class ="index">2</td>
+          <td>Panda</td>
+          <td>Cattee</td>
+          <td>4:06</td>
+          <td><Button>CATTY CLOUD SYNC</Button></td>
+        </tr>
+        <tr>
+          <td class ="index">3</td>
+          <td>Can't stop the Feeling!</td>
+          <td>Catin Climberlake</td>
+          <td>3:56</td>
+          <td><Button>CATTY CLOUD SYNC</Button></td>
+        </tr>
+        <tr>
+          <td class ="index">4</td>
+          <td>Work from Home</td>
+          <td>Cat Harmony feat Colla</td>
+          <td>3:34</td>
+          <td><Button>CATTY CLOUD SYNC</Button></td>
+        </tr>
+      </table>
+    </article>
+  </section>
+  <footer>
+    <div class="fchartimg">
+      <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" height=75em width=75em align: "left">
+    </div>
+    <p id="songName" align="left">If it ain't Love </p>
+    <p id="album" align="left">Catson Derulo</p>
+    <p align="right">
+      <Button><i class="la la-step-backward"></i></Button>
+      <Button><i class="la la-play"></i></Button>
+      <Button><i class="la la-step-forward"></i>
+      </Button> <Button><i class="la la-undo-alt"></i></Button>
+      <Button><i class="la la-random"></i></Button>
+    </p>
+  </footer>
+</body>
+</html>
```

Comment:
Please go through flexbox and floats and have a good understanding of those concepts... 
That will make this work much easier for you.
That is the only problem I see here. Otherwise, you were able to achieve 70-80% of the design target. Nice work!

Reply:
Try using "Colorpick Eyedropper" addon on Chrome/Edge browsers to have exact match with colors and not guess.